src/checkers/ai.rs

```diff
@@ -52,6 +68,35 @@ impl JumpMove {
 	fn with_jumps(from_row : usize, from_col : usize, jumps : Vec<JumpMove>) -> JumpMove {
 		JumpMove{ from_row : from_row, from_col : from_col, jumps : jumps }
 	}
+	
+	pub fn from_row(&self) -> usize {
+		self.from_row
+	}
+	
+	pub fn from_column(&self) -> usize {
+		self.from_col
+	}
+	
+	pub fn jumps(&self) -> &Vec<JumpMove> {
+		&self.jumps
+	}
+	
+	pub fn contains_jump_sequence(&self, jumps : &[BoardPosition]) -> bool {
+		if jumps.len() == 0 {
+			true
```

Comment:
This isn't exactly a base case, since it isn't used in a recursive function. However, checking this case now saves doing the check in the recursive function. I could have put this check in the recursive function instead, but it would only ever happen on the first invocation, so it's a slight optimization to check it here instead.